mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/pdfgen/PdfGenClient.kt

```diff
@@ -103,3 +133,37 @@ class PdfGenClient(
         else -> utbetaling.beregning
     }
 }
+
+@Serializable
+data class UtbetalingsdetaljerPdf(
+    val utbetaling: UtbetalingPdfDto,
+)
+
+@Serializable
+data class UtbetalingPdfDto(
```

Comment:
Bør vi ha helt egne typer for pdf-dataene? Og ikke dra inn Utbetaling.Tiltakstype etc...?

Reply:
Kanskje helst ja? Akkurat det som brukes i pdf'en